lib/test.js

```diff
@@ -40,7 +40,8 @@ class ExecutionContext extends assert.Assertions {
 				return test.compareWithSnapshot(options);
 			},
 			powerAssert: test.powerAssert,
-			experiments: test.experiments
+			experiments: test.experiments,
+			disableSnapshots: test.isHook === true
```

Comment:
This is a breaking change. We need to [add an experiment](https://github.com/avajs/ava/blob/master/lib/load-config.js#L10) for this and then _also_ check if that's been set in `test.experiments`.

Reply:
Addressed